File: rb/test/exists_test.rb
# DungeonsAndDragonsTwo SDK exists test

require "minitest/autorun"
require_relative "../DungeonsAndDragonsTwo_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = DungeonsAndDragonsTwoSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- DungeonsAndDragonsTwo SDK exists test

local sdk = require("dungeons-and-dragons-two_sdk")

describe("DungeonsAndDragonsTwoSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
